redfish/openapi/merged/redfish-openapi.yaml

```diff
@@ -36,6 +36,53 @@ paths:
           description: Error condition
       security:
       - {}
+  /redfish/v1/odata:
```

Comment:
move this block toward the end of the file i.e. after /redfish/v1/Systems/{ComputerSystemId} to align with the ordering in the original spec.